site/scripts/prepare-blocks.ts

```diff
@@ -154,37 +148,42 @@ const ensureRepositorySnapshot = async ({
     return repositorySnapshotDirPath;
   }
 
-  const { path: unzipDirPath, cleanup: cleanupUnzipDirPath } = await tmp.dir({
+  const { path: untarDirPath, cleanup: cleanupUntarDirPath } = await tmp.dir({
```

Comment:
```suggestion
  const { path: tarDirPath, cleanup: cleanupTarDirPath } = await tmp.dir({
```

`unzip` was chosen after the `unzip` command. Since we call `tar` instead of `untar`, maybe we don’t need `untar` as a term anywhere?